R/model_options.R

```diff
@@ -52,6 +54,7 @@ do_endpoint_model_options <- function(shape, survey, programme, anc) {
     area_level_options = area_level_options,
     area_level_default = area_level_options[[length(area_level_options)]]$id,
     calendar_quarter_t1_options = time_options,
+    calendar_quarter_t1_default = most_recent_survey_quarter,
```

Comment:
I think we need to make sure that `most_recent_survey_quarter` is in the list of `time_options` it probably always is? Time options is a fixed list of quarters from `Jan 2010` to now. I think putting a line to add `most_recent_survey_quarter` to the `time_options` if it isn't already in there would be worth while

Reply:
Thanks good shout. Addressed in 1505cc6.